README.md

```diff
@@ -3,129 +3,123 @@
+* Multiple files are transferred at once with an overall progress indicator.
 
 ### Send Files or Text Directly From Share Menu, Context Menu or CLI
-* [Send files directly from context menu on Windows](docs/how-to.md#send-files-directly-from-context-menu-on-windows)
-* [Send directly from share menu on iOS](docs/how-to.md#send-directly-from-share-menu-on-ios)
-* [Send directly from share menu on Android](docs/how-to.md#send-directly-from-share-menu-on-android)
-* [Send directly via command-line interface](docs/how-to.md#send-directly-via-command-line-interface)
-
-### Other changes
-* Change your display name permanently to easily differentiate your devices
-* [Paste files/text and choose the recipient afterwords ](https://github.com/RobinLinus/snapdrop/pull/534)
+* [Send files directly from the context menu on Windows](docs/how-to.md#send-files-directly-from-context-menu-on-windows)
+* [Send directly from the "Share" menu on iOS](docs/how-to.md#send-directly-from-share-menu-on-ios)
+* [Send directly from the "Share" menu on Android](docs/how-to.md#send-directly-from-share-menu-on-android)
+* [Send directly via the command-line interface](docs/how-to.md#send-directly-via-command-line-interface)
+
+### Other Changes
+* Change your display name to easily differentiate your devices.
+* [Paste files/text and choose the recipient afterwards ](https://github.com/RobinLinus/snapdrop/pull/534)
 * [Prevent devices from sleeping on file transfer](https://github.com/RobinLinus/snapdrop/pull/413)
 * Warn user before PairDrop is closed on file transfer
 * Open PairDrop on multiple tabs simultaneously (Thanks [@willstott101](https://github.com/willstott101))
-* [Video and Audio preview](https://github.com/RobinLinus/snapdrop/pull/455) (Thanks [@victorwads](https://github.com/victorwads))
-* Switch theme back to auto/system after darkmode or lightmode is enabled
+* [Video and audio preview](https://github.com/RobinLinus/snapdrop/pull/455) (Thanks [@victorwads](https://github.com/victorwads))
+* Switch theme back to auto/system after dark or light mode is on
 * Node-only implementation (Thanks [@Bellisario](https://github.com/Bellisario))
-* Automatic restart on error (Thanks [@KaKi87](https://github.com/KaKi87))
+* Auto-restart on error (Thanks [@KaKi87](https://github.com/KaKi87))
 * Lots of stability fixes (Thanks [@MWY001](https://github.com/MWY001) [@skiby7](https://github.com/skiby7) and [@willstott101](https://github.com/willstott101))
 * To host PairDrop on your local network (e.g. on Raspberry Pi): [All peers connected with private IPs are discoverable by each other](https://github.com/RobinLinus/snapdrop/pull/558)
-* When hosting PairDrop yourself you can [set your own STUN/TURN servers](docs/host-your-own.md#specify-stunturn-servers)
-* Built-in translations
+* When hosting PairDrop yourself, you can [set your own STUN/TURN servers](docs/host-your-own.md#specify-stunturn-servers)
+* Translations.
```

Comment:
I mentioned the "built-in" as it was also tried to add translations via the [snapdrop-android](https://github.com/fm-sys/snapdrop-android) app via JavaScript injections. Built-in translations are more stable, obviously.

Reply:
I think "translations" this is a long-term solution.